Automattic/wp-calypso
# Update LineChart Axes

This pull request refines axes of the new `LineChart` component in order to bring them closer to the ones used for bar charts on the regular `Stats` page (less tick density, integers only on the Y axis, different color scheme for tick labels ...):

Before | After
------ | -----
<img width="441" alt="screenshot" src="https://user-images.githubusercontent.com/594356/39672740-6ab1b946-5130-11e8-877f-0685c54ff37a.png"> | <img width="433" alt="screenshot" src="https://user-images.githubusercontent.com/594356/39672744-7d40caca-5130-11e8-8844-6c113d62072a.png">

Before | After
------ | -----
<img width="433" alt="screenshot" src="https://user-images.githubusercontent.com/594356/39672750-9add97b6-5130-11e8-97fc-6d97950d762c.png"> | <img width="422" alt="screenshot" src="https://user-images.githubusercontent.com/594356/39672754-a8f1d902-5130-11e8-955e-77129127330c.png">

## Testing

- Navigate to http://calypso.localhost:3000/devdocs/design/line-chart
- Click "Show Data Controls" and play around with the data. Try to create situations where the chart might misbehave and see how it handles it.
